backend/src/main/java/com/example/skicurort/curort/Curort.java

```diff
@@ -1,3 +1,36 @@
 package com.example.skicurort.curort;
 
-public class Curort {}
+import javax.persistence.*;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Entity
+@NoArgsConstructor
+@Table(name = "curort")
+public class Curort {
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private Long id;
+
+  @Column(name = "curort_name")
+  private String curortName;
+
+  @Column(name = "curort_address")
+  private String curortAdress;
+
+  @Column(name = "curort_email")
+  private String currortEmail;
+
+  @Column(name = "curort_phone")
+  private Long curortPhonenumber;
+
+  // TODO adding relation to person
+  // @OneToMany
+  // private <List> Person;
+  // TODO adding relation to person
+  // @OneToMany
+  // private <List> Shop;
```

Comment:
usuń zakomentowany kod przed commitem

Reply:
Troche niewiedziałem czy wogole to wstawiać a zakomentowane bo jeszcze nie ma tych klas. Jak to najlepiej robić jeżeli mam jakieś połaczenie do rzeczy które jeszcze nie zostały zrobione przez kogoś. A chciałbym aby juz to było przygotowane

Reply:
ok, jak uważasz